src/modules/integrator-guide/pages/compose-configuration/index.adoc

```diff
@@ -864,3 +864,37 @@ You can use this to create context-aware validators.
 ----
 length(title) < 5
 ----
+
+== Pageblock Refresh
+Pageblocks have a refresh option to automatically renew the pageblock data without having to reload the entire page. 
+This is useful for pageblocks that display data that changes frequently, such as a list of recent records, an update to a chart, or a list of recent comments.
+
+=== Pageblocks With Refresh
+Not all pageblocks have the refresh option enabled. Below is a list of pageblocks that have the refresh option enabled.
+
+* Record List
+* Chart
+* Comment
+* Iframe
+* Metric
+* Progress
+* Record Organizer
+* Report
+* Social Feed
+* Calendar
+
+=== Enabling Pageblock Refresh
+To enable refreshing on a pageblock, you will need to find the refresh option in the *general tab* when you edit the pageblock. 
+image:compose-configuration/page-block-general.png[Pageblock General Tab, width=49%]
```

Comment:
If you do 

```
text
image
text
```

it will inline the image with the text.

![image](https://user-images.githubusercontent.com/12545711/232439627-402d63cc-08f0-4b6a-92e2-2ee6cb17309a.png)

If you do 

```
text

image

text
```

it will make a _new paragraph_ for the image.

![image](https://user-images.githubusercontent.com/12545711/232440069-08510610-566a-4d93-a806-4193634cd2ad.png)

___

You can use inlined images; I did it here for example

![image](https://user-images.githubusercontent.com/12545711/232440024-1cfecc19-b148-484f-b006-ccbe5bd6a8a0.png)

You can also add `role="data-zoomable"` (`image::compose-configuration/field-validation.png[role="data-zoomable"]`) to make it clickable and expandable like so:

![image](https://user-images.githubusercontent.com/12545711/232440262-21175829-58ee-4bf2-bbfc-2864728bd0df.png)